solr/modules/opentelemetry/src/test/org/apache/solr/opentelemetry/TestDistributedTracing.java

```diff
@@ -201,7 +201,7 @@ private void assertOneSpanIsChildOfAnother(List<SpanData> finishedSpans) {
     assertEquals(child.getTraceId(), parent.getTraceId());
   }
 
-  private List<SpanData> getAndClearSpans() {
+  static List<SpanData> getAndClearSpans() {
```

Comment:
This is a hack, calling some method on some other test to manage global lifecycle of some singleton mock tracer.  Seems like we might ideally have a JUnit Rule to manage trace listening, where we might have this method?